install/cameras.csv

```diff
@@ -1,47 +1,47 @@
+Mobotix AG,M12 3MP,10.5.32.131,RIMM,01,0,270,0,0,0,2010-10-29T01:00:00Z,2014-10-27T03:40:00Z,
+Mobotix AG,M12 3MP,10.5.32.131,RIMM,01,0,270,0,0,0,2014-10-27T20:46:00Z,2017-09-24T00:00:00Z,
+Mobotix AG,M12 3MP,10.6.15.32,WINR,01,0,0,1.5,0,0,2013-04-23T01:00:00Z,9999-01-01T00:00:00Z,White Island North Rim looking into the crater
+Mobotix AG,M15,10.13.10.47,WHWI,01,0,19,-4,0,0,2013-08-07T00:00:00Z,9999-01-01T00:00:00Z,Whakatane looking at White Island
+Mobotix AG,M15,10.13.68.240,WIWR,01,0,0,1.5,0,0,2013-12-10T00:00:00Z,2016-09-14T00:00:00Z,
+Mobotix AG,M15,10.13.68.240,N,01,0,0,0,0,0,2017-09-12T00:00:00Z,2019-05-22T21:50:00Z,Mountain Air (Whakapapa) looking at Ngauruhoe
+Mobotix AG,M15,10.14.178.98,RIMM,01,0,270,-1.5,0,0,2017-09-24T00:00:00Z,9999-01-01T00:00:00Z,Mt Moumoukai (Raoul Island) looking at Green Lake
+Mobotix AG,M15,10.16.105.12,TOPI,01,0,0,2,0,0,2015-05-28T00:01:00Z,2019-03-04T20:30:00Z,
+Mobotix AG,M15,10.16.105.12,N,01,0,87,4,0,0,2019-05-23T00:00:00Z,9999-01-01T00:00:00Z,Mountain Air (Whakapapa) looking at Ngauruhoe
+Mobotix AG,M15,10.17.107.14,MTSR,01,0,0,2,0,0,2016-02-02T02:00:00Z,9999-01-01T00:00:00Z,Mangateitei looking at South Ruapehu
+Mobotix AG,M15,10.18.101.137,CVTP,01,10,290,-2.5,0,0,2016-07-11T00:03:00Z,2019-06-02T23:40:00Z,Tai Ping looking at East Ruapehu & South East Ngauruhoe
+Mobotix AG,M15,10.18.101.137,CVTP,01,-8,290,-2.5,0,0,2019-06-02T23:50:00Z,2019-06-19T22:00:00Z,Tai Ping looking at East Ruapehu & South East Ngauruhoe: knocked off-alignment by ice
+Mobotix AG,M15,10.18.101.137,CVTP,01,10,290,-2.5,0,0,2019-06-19T22:00:00Z,2019-07-06T21:10:00Z,Tai Ping looking at East Ruapehu & South East Ngauruhoe
+Mobotix AG,M15,10.18.101.137,CVTP,01,4,290,-2.5,0,0,2019-07-06T21:20:00Z,2019-07-08T00:40:00Z,Tai Ping looking at East Ruapehu & South East Ngauruhoe
+Mobotix AG,M15,10.18.101.137,CVTP,01,0.5,290,-2.5,0,0,2019-07-08T00:50:00Z,2019-11-27T20:50:00Z,Tai Ping looking at East Ruapehu & South East Ngauruhoe
+Mobotix AG,M15,10.18.101.137,CVTP,01,9.5,290,-2.5,0,0,2019-11-27T21:00:00Z,2020-05-07T00:19:59Z,Tai Ping looking at East Ruapehu & South East Ngauruhoe
+Mobotix AG,M15,10.18.101.33,WIWR,01,0,0,1.5,0,0,2016-10-03T06:14:00Z,9999-01-01T00:00:00Z,White Island West Rim looking into the crater
+Mobotix AG,M15,10.18.168.61,CVWF,01,0,0,0,0,0,2016-06-01T23:40:00Z,9999-01-01T00:00:00Z,White Island Factory looking across the crater floor
+Mobotix AG,M15,10.20.210.120,CVTP,01,9.5,290,-2.5,0,0,2020-05-07T00:20:00Z,9999-01-01T00:00:00Z,Tai Ping looking at East Ruapehu & South East Ngauruhoe
+Mobotix AG,M15,10.20.245.115,TOTM,01,0,0,0,0,0,2018-05-03T00:00:00Z,9999-01-01T00:00:00Z,Karewerewa looking at TeMaari
+Mobotix AG,M15,10.7.46.153,TOTM,01,0,0,0,0,0,2017-10-12T00:40:00Z,2018-05-02T23:55:00Z,
+Mobotix AG,M24M-Secure,10.9.190.163,TCNW,01,0,315,0,0,0,2011-11-28T00:00:00Z,2019-02-24T21:00:00Z,
+Mobotix AG,M24M-Secure,10.9.190.76,TCNE,02,0,45,0,0,0,2011-11-28T00:00:03Z,2019-02-24T21:00:00Z,
```

Comment:
How can this be view 02 if view 01 is not in the table?

Reply:
I just made these up so may not be real life, clearly needs a test to check this.

Reply:
The other thing is I haven't changed the name of the camera in this instance ( tcne should be something else with 01 02 03 etc)